apps/govern/components/Proposals.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Card, Col, Row, Typography } from 'antd';
+import Image from 'next/image';
+import styled from 'styled-components';
+
+import { UNICODE_SYMBOLS } from 'libs/util-constants/src';
+
+const StyledMain = styled.main`
+  display: flex;
+  flex-direction: column;
+  max-width: 946px;
+  margin: 0 auto;
+`;
+
+const Title = styled.h1`
+  font-size: 24px;
+  margin: 0 0 8px;
+`;
+
+const { Paragraph } = Typography;
+
+type Proposal = {
+  title: string;
+  description: string;
+  imageSrc: string;
+  button: {
+    title: string;
+    href: string;
+  };
+};
+const proposals: Proposal[] = [
+  {
+    title: 'Snapshot proposals',
+    description: 'Proposals that are made and enacted off-chain.',
+    imageSrc: '/images/snapshot.png',
+    button: {
+      title: 'Vote on Snapshot',
+      href: 'https://snapshot.org/#/autonolas.eth',
+    },
+  },
+  {
+    title: 'Boardroom proposals',
+    description: 'Proposals that are made and enacted on-chain.',
```

Comment:
description here is the same as above, unsure if this is intended?

Reply:
no, the on is "off-chain" and this one if "on-chaine"